src/pages/dashboard.tsx

```diff
@@ -73,6 +75,14 @@ const Dashboard = () => {
     }
   };
 
+  const logout = () => {
+    signOut()
+      .then(() => {
+        void router.push("/login");
```

Comment:
I feel like it should go to the home page lol
```suggestion
        void router.push("/");
```